autoflip_calculators/text_detection_calculator_test.cc

```diff
@@ -78,7 +78,9 @@ CalculatorGraphConfig::Node MakeConfig(std::string base_config, bool use_visual_
   config.mutable_options()
       ->MutableExtension(TextDetectionCalculatorOptions::ext)
       ->set_use_visual_scorer(use_visual_scorer);
-
+  config.mutable_options()
+      ->MutableExtension(TextDetectionCalculatorOptions::ext)
+      ->set_model_path("/usr/local/google/home/zzhencchen/mediapipe/mediapipe/models/frozen_east_text_detection.pb");
```

Comment:
I don't think this unit test will pass on a server.  We need to figure out some way to have a rel model path or just include a copy of the model path in the CL maybe?